# HG changeset patch
# User Sergey Mironov <grrwlf@gmail.com>
# Date 1381747050 -7200
#      Mon Oct 14 12:37:30 2013 +0200
# Node ID 1f44f5a8f232b1881b8ad6b5dfaf4f5e724bbcba
# Parent  338e6357ebd61432dfb85ca2f7612dc7ec40c8c6
CYGWIN HACK: disable stack size option

diff --git a/src/c/request.c b/src/c/request.c
--- a/src/c/request.c
+++ b/src/c/request.c
@@ -127,6 +127,7 @@
 
 int pthread_create_big(pthread_t *outThread, void *foo, void *threadFunc, void *arg)
 {
+/*
   int err;
   pthread_attr_t stackSizeAttribute;
 
@@ -137,8 +138,9 @@
     err = pthread_attr_setstacksize(&stackSizeAttribute, stackSize);
     if (err) return err;
   }
+*/
 
-  return pthread_create(outThread, &stackSizeAttribute, threadFunc, arg);
+  return pthread_create(outThread, NULL, threadFunc, arg);
 }
 
 void uw_request_init(uw_app *app, void *logger_data, uw_logger log_error, uw_logger log_debug) {
